utils/__init__.py

```diff
@@ -15,3 +16,18 @@ def create_allure_environment_opts(opts: dict, dst: tp.Optional[pathlib.Path] =
             )
         )
         file.write("\n")
+
+
+def setup_logging(log_level=logging.DEBUG):
+    """Setup root logger and quiet some levels."""
+    logger = logging.getLogger()
+    logger.setLevel(log_level)
+
+    logging.getLogger("web3.RequestManager").setLevel(logging.WARNING)
```

Comment:
pls add comment why you change this level for web3

Reply:
Added